lib/db/event-store.js

```diff
@@ -27,15 +31,26 @@ class EventStore extends EventEmitter {
             .from('events')
             .limit(100)
             .orderBy('created_at', 'desc')
+            .orderBy('id', 'desc') // When multiple events have the same timestamp, assume later ids are newer
             .map(this.rowToEvent);
     }
 
     getEventsFilterByName(name) {
+        let whereClause;
```

Comment:
as with https://github.com/Unleash/unleash/pull/450/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.lock#r295445476

Is there something protecting this from being vulnerable to injection btw? They all have params apart from sqlite3 which seems to string interpolate.

Reply:
Yes, the potential vulnerability is only for sqlite3. This is called out in the PR as being one of the big reasons not to merge it :). If you can find the correct knex syntax to do a like comparison with a bound parameter, I'm totally willing to integrate it.

Reply:
https://github.com/tgriesser/knex/pull/2814/files seems to be where JSON column support was added. It's not so long ago, so maybe SQLite support in this PR is just not a great thing to have. (IDK)